package.json

```diff
@@ -478,6 +478,11 @@
           "default": true,
           "description": "Enables syntax based code folding. When disabled, the default indentation based code folding is used."
         },
+        "powershell.codeFolding.showLastLine": {
+          "type": "boolean",
+          "default": false,
```

Comment:
@tylerl0706 @rkeithhill @glennsarti @RussPitcher Thoughts on this being the default? (Note as in the description below that this is VSCode's default)